### PR TITLE
feat: Show Main medical condition from subpopulations on main cohort page under "General design"?

### PR DESCRIPTION
**Pull request by connoratrug**
Show Main medical condition from subpopulations on main cohort page under "General design"? 

https://trac.molgeniscloud.org/ticket/10601

🚨 note pref impact when fetching large ontology trees ( subpopulations * tree ), if n-subpopulations is large and or tree is deep

### CODE REVIEW

**Review by mswertz — COMMENTED** [review]
Code ziet er goed uit, maar op 'acc' zie je dat 'Oncolifes' meer dan 10sec duurt om op te halen. Dat lijkt me teveel. Dus ik denk belangrijk dat je contact opneemt met CBCH om deze PR af te raden? (wel de refactorings doorvoeren uit deze PR).

File: apps/nuxt3-ssr/nuxt.config.ts [hunk 1]
```diff
@@ -29,6 +29,12 @@ const config: NuxtConfig = {
   nitro: {
     compressPublicAssets: { brotli: true },
   },
+  imports: {
+    transform: {
+      // exclude
+      exclude: [/\bmeta-data-utils\b/],
```
[the remark this answers is not in the document]
Comment (connoratrug) [reply apps/nuxt3-ssr/nuxt.config.ts:35]: yea still having problem duren dev where nuxt tries to recompile the already compiled lib module , i'll remove it from this pr

File: apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/[catalogue]/cohorts/[cohort]/index.vue [hunk 2]
```diff
@@ -131,6 +131,12 @@ const query = gql`
       datasets {
         name
       }
+    }
+     Subcohorts(
+      filter: { resource: { id: { equals: [$id] } },  }
+    ) {
+      name
+      mainMedicalCondition ${moduleToString(ontologyFragment)}
```
[the remark this answers is not in the document]
Comment (connoratrug) [reply apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/[catalogue]/cohorts/[cohort]/index.vue:139]: yes i suspect is the 'ontologyFragment' part it expands to parent.parent.partent..... ( 9 times) , if you tree is deeper you just get more 'roots' i think

File: apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/[catalogue]/cohorts/[cohort]/index.vue [hunk 3]
```diff
@@ -141,28 +147,49 @@ const query = gql`
...
+  if (!subcohorts.value || !subcohorts.value.length) {
+    return [];
+  } else {
+    return subcohorts.value
+      .map((s: { mainMedicalCondition?: IOntologyItem[] }) => {
+        const combinedItems = s.mainMedicalCondition
+          ? s.mainMedicalCondition
+          : [];
+
+        return combinedItems as IOntologyItem[];
```
[the remark this answers is not in the document]
Comment (connoratrug) [reply apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/[catalogue]/cohorts/[cohort]/index.vue:183]: the build tree function removes the duplicates, but could do it before we call the buildTree function here to be more specific

**Review by mswertz — APPROVED** [review]

**Pull request merged**